2-Browsers/Week1/homework/ex4-whatsTheTime/index.js

```diff
@@ -7,6 +7,60 @@
 ------------------------------------------------------------------------------*/
 function addCurrentTime() {
   // TODO complete this function
+  const date = new Date();
+  let h = date.getHours();
+  let m = date.getMinutes();
+  let s = date.getSeconds();
+  let session = 'AM';
+
+  if (h === 0) {
+    h = 12;
+  }
+
+  if (h > 12) {
+    h = h - 12;
+    session = 'PM';
+  }
+
+  h = h < 10 ? '0' + h : h;
+  m = m < 10 ? '0' + m : m;
+  s = s < 10 ? '0' + s : s;
+
+  const time = h + ':' + m + ':' + s + ' ' + session;
+
+  const divClockContainer = document.getElementById('divClockContainer');
+  divClockContainer.innerText = time;
 }
 
 // TODO execute `addCurrentTime` when the browser has completed loading the page
+
+function createDom() {
+  const googleFont = document.createElement('link');
+  googleFont.setAttribute(
+    'href',
+    'https://fonts.googleapis.com/css2?family=Orbitron&display=swap'
+  );
+  googleFont.setAttribute('rel', 'stylesheet');
+  document.head.appendChild(googleFont);
+  const body = document.querySelector('body');
+  body.style.background = 'black';
+  const divClockContainer = document.createElement('div');
+  divClockContainer.id = 'divClockContainer';
+  divClockContainer.style.position = 'absolute';
+  divClockContainer.style.top = '50%';
+  divClockContainer.style.left = '50%';
+  divClockContainer.style.transform = 'translateX(-50%) translateY(-50%)';
+  divClockContainer.style.color = '#17D4FE';
+  divClockContainer.style.fontSize = '60px';
+  divClockContainer.style.fontFamily = 'Orbitron';
+  divClockContainer.style.letterSpacing = '7px';
+  body.appendChild(divClockContainer);
+}
+
+function initialize() {
+  createDom();
+  addCurrentTime();
+  setInterval(addCurrentTime, 1000);
+}
+
+window.addEventListener('load', initialize);
```

Comment:
Your clock looks nice! But I now see that it's not completely your own design.

Reply:
I thought I could use the internet by mention the references.